packages/ndla-ui/src/Article/ArticleSideBar.tsx

```diff
@@ -74,7 +74,7 @@ const ArticleSideBar = ({
       //TOP
       if (resourcesRef.current.getBoundingClientRect().top <= 0) {
         setHide(true);
-      } else if (resourcesRef.current.getBoundingClientRect().top <= 0) {
+      } else if (resourcesRef.current.getBoundingClientRect().top >= 0) {
```

Comment:
Denne var med vilje?

Reply:
Ja. Man får linting-feil dersom det er to like regler på rad. Dette sørger for at "hopp til ressurser"-greia på tverrfaglige temaer forsvinner når man kommer til ressursene. Fungerte ikke i det hele tatt før.